packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -13,6 +13,7 @@ import * as fs from 'fs';
 import * as os from 'os';
 import * as semver from 'semver';
 import * as _ from 'lodash';
+import * as findUp from 'find-up';
```

Comment:
find-up isn't a dependency of @react-native-windows/cli currently.  You'll either need to add it, or use something else.